apps/src/templates/studioHomepages/TeacherHomepage.story.jsx

```diff
@@ -38,6 +38,7 @@ const serverCourses = [
 
 export default {
   component: TeacherHomepage,
+  decorators: [withGlobalEdition],
```

Comment:
This storybook decorator responses to "region" being specified in the story template.